package.json

```diff
@@ -69,5 +69,8 @@
   "devDependencies": {
     "eslint": "^7.24.0",
     "prettier": "^2.2.1"
+  },
+  "prettier": {
+    "parser": "babel"
```

Comment:
What's the reason for this? It shouldn't be necessary to specify the parser.

Reply:
Sorry, this is a misconfiguration on my part. I had an issue where Prettier was very aggresive in creating newlines in code, but this eventually turned out to be a problem in my IDE configuration - so I removed this line and reformatted the .tsx file in a new commit.